pkgs/tools/security/regpg/default.nix

```
@@ -0,0 +1,56 @@
+{ lib
+, stdenv
+, fetchgit
+, git
+, perl
+, perlTextMarkdown
+}:
+
+stdenv.mkDerivation rec {
+  pname = "regpg";
+  version = "1.11";
+
+  src = fetchgit {
+    url = "https://github.com/fanf2/regpg.git";
+    rev = "refs/tags/regpg-${version}";
+    sha256 = "9492edfa89d044c37a72f48e5753c0054fdf7f1100c834c8eadf518ce93d0d1c";
+    leaveDotGit = true;
+  };
+
+  builder = builtins.toFile "builder.sh" ''
+    source $stdenv/setup
+
+    export PATH="$git/bin:$perl/bin:$PATH"
+    export PERL5LIB="$perlTextMarkdown/lib/perl5/site_perl"
+
+    cp -r $src ./regpg
+    chmod -R +w regpg
+    cd regpg
+
+    # Switch to nix perl (for build scripts)
+    patchShebangs --build ./
+
+    # Hack to make `get describe` return the version tag
+    # since tags are deleted by fetchgit (?)
+    git config user.name "nobody"
+    git config user.email "nobody@example.com"
+    git tag -a "regpg-$version" -m "regpg-$version"
+
+    make
+
+    # Switch to nix perl (for executable)
+    patchShebangs --build ./regpg
+
+    make prefix="" DESTDIR=$out install
+  '';
+
+  meta = with lib; {
+    description = "GPG wrapper utility for storing secrets in VCS";
+    homepage    = "https://dotat.at/prog/regpg/";
+    license     = licenses.gpl3;
+    platforms   = platforms.all;
+    maintainers = with maintainers; [ "0xc45" ];
```

Add yourself to `maintainers/maintainer-list.nix`, and then:
```suggestion
    maintainers = with maintainers; [ 0xc45 ];
```

---

more like
```suggestion
    maintainers = with maintainers; [ _0xc45 ];
```
and needs an addition to `maintainers.nix`